### PR TITLE
Viewing rooms home rail

### PR DESCRIPTION
**Pull request by pvinis**
~⚠️ Merge **after** #3595 ⚠️~

The type of this PR is: **Feature**

This PR resolves **[GALL-2879]**

### Description

Adding a rail in Home, showing the featured VRs, and linking to the VR landing page.

Should we have this as a module of `homePage` in MP 🤔? Why? Why not?

### Test Plan

- enable viewing rooms flag in dev menu
- be in home page

### Screenshots

Screenshot 2020-07-22 at 14 35 07

[GALL-2879]: https://artsyproduct.atlassian.net/browse/GALL-2879

### CODE REVIEW

**Comment by ds300** [comment]
Hmm I noticed this is using `View all` as a CTA, instead of the chevron like all the other rails on the home page. @samchieng should that be changed?

**Comment by pvinis** [comment]
I am waiting for feedback from @tinn for the same.

**Comment by pvinis** [comment]
After talking to the designers, the decision is:
- We use `SectionTitle`.
- We use `View all` (sans3) instead of `>`.

**Comment by pvinis** [comment]
This is ready to go too 👍

File: src/lib/Components/SectionTitle.tsx [hunk 1]
```diff
@@ -14,11 +14,12 @@ const Wrapper: React.FC<{ onPress?(): any }> = ({ onPress, children }) => {
...
-export const SectionTitle: React.FC<{ title: React.ReactNode; subtitle?: React.ReactNode; onPress?: () => any }> = ({
-  title,
-  subtitle,
-  onPress,
-}) => {
+export const SectionTitle: React.FC<{
+  title: React.ReactNode
+  subtitle?: React.ReactNode
+  onPress?: () => any
+  RightButtonContent?: React.ComponentType<any> | null
```
Comment (ds300) [inline src/lib/Components/SectionTitle.tsx:21]: 👍

**Pull request merged**